### PR TITLE
Add card component

### PR DESCRIPTION
**Pull request by xsharonhe**
Screen Shot 2021-07-20 at 1 04 29 PM

### CODE REVIEW

File: components/Card.jsx [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    </Wrapper>
+  );
+}
+
+const Wrapper = styled.div`
+  margin-left: 20px;
+  margin-top: 20px;
+  border-radius: 44px;
+  margin-bottom: 30px;
+  max-width: 300px;
```
Comment (xsharonhe) [inline components/Card.jsx]: I need to change this to imageWidth later.

**Comment by hirundinidae** [comment]
Looks really good! Just two things I noticed: 
1. on wider screens the cards are really far apart 
image
- I think at either 1500/1600 width, split to 4 cards per slide 
image
  - but, this just keeps expanding with wider screens (at even wider, -> 5 per slide, 6 per slide, etc) 
- or cap the width of the carousel with 3/slide (when the screen is ~1200) 
2. (smaller thing) Slightly confused at first because I thought the carousel would auto move on to the next slide since 
  a. there were no arrows
  b. not immediately clear that I can swipe/drag the cards to the next slide 
  c. didn't realize clicking the circles at the bottom also moves the cards 
- maybe add arrows on the side of the cards (click -> next slide) like the mock 
- small arrows or text near the 3 dots 
- not really sure, but if you can think of anything to indicate `swipe to see more` 

Otherwise, it works great on average and smaller screens!

File: components/CardCarousel.jsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+          clickable: true,
+        }}
+        breakpoints={{
+          1200: {
+            slidesPerView: 3,
+            spaceBetween: 20,
+          },
+          750: {
+            slidesPerView: 2,
+            spaceBetween: 20,
```
Comment (hirundinidae) [inline components/CardCarousel.jsx:33]: (For 1. from the review comment, add more slidesPerView as the screen gets wider): 
- continue increasing slidesPerView relative to the screen width (eg. 1500-1600 for 4 per view, etc) 
  - might be nice to keep the width of the carousel at 100% if there's an easy way to keep increasing slidesPerView as screen increases

File: sections/home/WhatsHappening.jsx [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+const Wrapper = styled.div`
+  margin: 0 5vw 5vh 5vw;
+`;
+const CarouselWrapper = styled.div`
+  margin: auto;
+  ${media(
+    1200,
+    `
+            width: 70%;
+        `
```
Comment (hirundinidae) [inline sections/home/WhatsHappening.jsx]: (for 1. from the review comment, cap the width size) 
- I *think* this is where carousel width is set 
  - set a constant width after some width
  - eg. if max 3 slides per view, then after 1400 (or other), the carousel will stay the same width and avoid really far apart cards

**Review by hirundinidae — APPROVED** [review]
Looks great!

**Pull request merged**